.github/workflows/notion.yml

```diff
@@ -18,20 +18,8 @@ jobs:
         # with:
           # ref: ${{ github.head_ref }}
 
-      - name: notion-blog
-        uses: xzebra/notion-blog@master
-        with:
-          database-id: 'f887dfac795547ff97a81bb669b1052f'
-          content-folder: web/content
-          images-folder: web/static/images/posts
-          images-link: /images/posts
-          archetype-file: web/themes/bookworm-light/archetypes/blog.md
-          property-description: "Description"
-          property-tags: "Tags"
-          property-categories: "Categories"
-          filter-prop: 'Status'
-          filter-value: 'Finished ✅'
-          published-value: 'Published 🖨'
```

Comment:
This is a very common way of configuring Github Actions and forcing to use a json file in the repository just limits the users.

Reply:
I know this is a very common way, but in the previous code, when I have a configuration file, I still have to define it once in yaml, which is equivalent to two configurations.

To avoid disruptive changes, we can support both two ways, flag first. Do you think it's okay?